momento-sdk/src/main/java/momento/sdk/ScsDataClient.java

```diff
@@ -3716,4 +3724,34 @@ private _DictionaryDeleteRequest.Some addSomeFieldsToRemove(@Nonnull List<ByteSt
   public void close() {
     scsDataGrpcStubsManager.close();
   }
+
+  public static void main(String[] args) {
```

Comment:
i presume this was just for dev and would be removed

Reply:
It was. I left it in just to show a more simple example of how the thread pool limits the concurrent requests.